portfolio/src/main/java/com/google/sps/data/TaskQueue.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.data;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+import java.util.PriorityQueue;
+
+/**
+ * This class holds a List of tasks and includes functions to sort
+ * the List and retrieve elements.
+ */
+public class TaskGroup {
```

Comment:
The file name is TaskQueue, but the class name is TaskGroup. Can we make the class name TaskQueue?

And we'll want to update the references to say TaskQueue instead of TaskGroup.

Reply:
Done.